src/MySql/custom/New-AzMySqlFlexibleServer.ps1

```diff
@@ -19,188 +19,589 @@ Creates a new MySQL flexible server
+            $null = $PSBoundParameters.Remove('AdministratorUserName')
+        }
+        Else {
+            $PSBoundParameters.AdministratorLogin = Get-RandomName
+        }
+
+        # Handling Vnet & Subnet
+        $NetworkKeys = 'PublicAccess', 'Subnet', 'Vnet', 'SubnetPrefix', 'VnetPrefix'
+        $NetworkParameters = @{}
+        ForEach($Key in $NetworkKeys){
+            If ($PSBoundParameters.ContainsKey($Key)){ 
+                $NetworkParameters[$Key] = $PSBoundParameters[$Key]
+                $null = $PSBoundParameters.Remove($Key)
             }
-            else {
-                $PSBoundParameters.StorageProfileBackupRetentionDay = 7
+        }
+        $RequiredKeys = 'SubscriptionId', 'ResourceGroupName', 'Name', 'Location'
+        ForEach($Key in $RequiredKeys){ $NetworkParameters[$Key] = $PSBoundParameters[$Key] }
+
+        If(!$NetworkParameters.ContainsKey('PublicAccess')){
+            $VnetSubnetParameters = CreateNetworkResource $NetworkParameters
+            $SubnetId = GetSubnetId $VnetSubnetParameters.ResourceGroupName $VnetSubnetParameters.VnetName $VnetSubnetParameters.SubnetName
+            $PSBoundParameters.DelegatedSubnetArgumentSubnetArmResourceId = $SubnetId
+            If ([string]::IsNullOrEmpty($PSBoundParameters.DelegatedSubnetArgumentSubnetArmResourceId)) {
+                $null = $PSBoundParameters.Remove('DelegatedSubnetArgumentSubnetArmResourceId')
             }
+        }
+
+        $Msg = 'Creating MySQL server {0} in group {1}...' -f $PSBoundParameters.Name, $PSBoundParameters.resourceGroupName
+        Write-Host $Msg
+        $Msg = 'Your server {0} is using sku {1} (Paid Tier). Please refer to https://aka.ms/mysql-pricing for pricing details' -f $PSBoundParameters.Name, $PSBoundParameters.SkuName
+        Write-Host $Msg
+        $Server = Az.MySql.internal\New-AzMySqlFlexibleServer @PSBoundParameters
+
+        # # Create Database
+        $DatabaseParameter = [Microsoft.Azure.PowerShell.Cmdlets.MySql.Models.Api20171201.Database]::new()
+        $DatabaseParameter.Charset = "utf8"
+        $DatabaseParameter.Collation = "utf8_general_ci"
+        $Msg = 'Creating database {0}...' -f $DEFAULT_DB_NAME
+        Write-Host $Msg
+        $null = New-AzMySqlFlexibleServerDatabase -Name $DEFAULT_DB_NAME -ResourceGroupName $PSBoundParameters.ResourceGroupName -ServerName $PSBoundParameters.Name -Parameter $DatabaseParameter
+        
+        # Create Firewallrules
+        $FirewallRuleName = CreateFirewallRule $NetworkParameters
+
+        $Msg = "
+`"databaseName`": `"$DEFAULT_DB_NAME`",
+`"id`": `"/subscriptions/$($PSBoundParameters.SubscriptionId)/resourceGroups/$($PSBoundParameters.ResourceGroupName)/providers/Microsoft.DBForMySql/flexibleServers/$($PSBoundParameters.Name)`",
+`"location`": `"$($PSBoundParameters.Location)`",
+`"password`": `"$($PSBoundParameters.AdministratorLoginPassword)`",
```

Comment:
For security issue, password cannot be printed as plain text. You should use security string.

Reply:
The password is autogenerated that we need to inform the users what password is. What is the best way to show it to the customers?